rudder-lang/src/ir/variable.rs

```diff
@@ -66,7 +66,16 @@ impl<'src> CondVariableDef<'src> {
             state_params,
         } = var;
         let (error, metadata) = create_metadata(metadata);
-        context.add_variable_declaration(name, Type::Enum(Token::new("corelib.rl", "boolean")))?;
+        context.add_variable_declaration(name, Type::Enum(Token::new("rudderlang", "boolean")))?;
+        // also add report_data to variables since these are generated and therefore their existence is checked at compile time
+        context.add_variable_declaration(
```

Comment:
create a variable for special cases (condition_from). Without these rudderc would return an error with undefined variable calls